examples/prisma-example/apps/api/src/resources/account/actions/google.ts

```diff
@@ -0,0 +1,79 @@
+import { userService } from 'resources/user';
+
+import { authService, googleService } from 'services';
+
+import config from 'config';
+
+import { AppKoaContext, AppRouter } from 'types';
+
+const getOAuthUrl = async (ctx: AppKoaContext) => {
+  const areCredentialsExist = config.GOOGLE_CLIENT_ID && config.GOOGLE_CLIENT_SECRET;
+
+  ctx.assertClientError(areCredentialsExist, {
+    global: 'Setup Google OAuth credentials on API',
+  });
+
+  ctx.redirect(googleService.oAuthURL);
+};
+
+const signInGoogleWithCode = async (ctx: AppKoaContext) => {
+  const { code } = ctx.request.query;
+
+  const { isValid, payload } = await googleService.exchangeCodeForToken(code);
+
+  ctx.assertError(isValid && payload && !(payload instanceof Error), `Exchange code for token error: ${payload}`);
+
+  const user = await userService.findUnique({
+    where: {email: payload.email},
```

Comment:
```suggestion
    where: { email: payload.email },
```

run 'pnpm prettier' to fix code style